File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        total = 0
        for right in range(len(arr)):
            if arr[right] % 2 != 1: total = 0
            else: total += 1
            if total == 3: return True
        return False

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        var counter = 0;
        for (var num : arr){
            if (num % 2 == 0) counter = 0;
            else counter += 1;
            if (counter == 3) return true;
        }
        return false;
    }
}